remote.lua

```diff
@@ -31,11 +31,76 @@ local function remote_rebuild(player_name)
     global.evogui[player.name].version = ""
 end
 
+--
+-- Creates a sensor managed by a remote interface (another mod or script)
+-- sensor_name: internal name of the sensor. Should be unique.
+-- sensor_text: Text to display in the active gui
+-- sensor_caption: Sensor setting name in the EvoGUI settings panel
+-- sensor_color: Font color of the text to display in the active gui, optional, may be nil
+-- example: remote.call("EvoGUI", "create_remote_sensor", "mymod_my_sensor_name", "Text: Lorem Ipsum", "[My Mod] Lorem Ipsum Text")
+local function create_remote_sensor(sensor_name, sensor_text, sensor_caption, sensor_color)
+    if not sensor_name then
+        evogui.log({"err_nosensorname"})
+        return
+    end
+
+    if not sensor_text then
+        evogui.log({"err_nosensortext"})
```

Comment:
Since we have a sensor name by this point, it would be good to include that in the error message (both here, and below).